NuKeeper.AzureDevOps/AzureDevOpsSettingsReader.cs

```diff
@@ -9,11 +9,37 @@ namespace NuKeeper.AzureDevOps
 {
     public class AzureDevOpsSettingsReader : ISettingsReader
     {
-        private readonly FileSettings _fileSettings;
+        public Platform Platform => Platform.AzureDevOps;
 
-        public AzureDevOpsSettingsReader(IFileSettingsCache settingsCache)
+        public bool CanRead(Uri repositoryUri)
         {
-            _fileSettings = settingsCache.GetSettings();
+            if (repositoryUri == null || repositoryUri.Host != "dev.azure.com")
+            {
+                return false;
+            }
+
+            return true;
+        }
+
+        public AuthSettings AuthSettings(Uri apiUri, string accessToken)
+        {
+            if (apiUri == null)
+            {
+                return null;
+            }
+
+            var testUri = UriFormats.EnsureTrailingSlash(apiUri);
+
+            if (!testUri.IsWellFormedOriginalString()
+                || (testUri.Scheme != "http" && testUri.Scheme != "https"))
+            {
+                return null;
+            }
+
+            var envToken = Environment.GetEnvironmentVariable("NuKeeper_azure_devops_token");
```

Comment:
This env var should go in the docs after merge. 
Or is it better to rename the env var to something more general, e.g. `nukeeper_auth_token`

Reply:
After discussion on Gitter will leave as is for now